MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift

```diff
@@ -12,14 +12,14 @@ import UIKit
 class AnimationTableViewController: UITableViewController {
     weak var delegate: AnimationTableDelegate?
 
-    private let data = ["Spread", "Row", "Column"]
+    private let animationTypeNames = AnimationTypes.getAllAnimationTypesNames()
```

Comment:
I moved the function `getAllAnimationTypesNames()` to a class called `AnimationManager`.
So this line should be
```
private let animationTypeNames = AnimationManager.instance.getAllAnimationTypesNames()
```
I'll merge this PR in now and make the change in another PR (sorry about the inconsistency)